src/Migrations/Version20200411130257.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
```

Comment:
Please remove this automatic comment

Reply:
We have removed both migrations after a discussion with @mRoca to avoid run both for no reason in production